src/OpenTelemetry.Instrumentation.Process/README.md

```diff
@@ -100,9 +100,9 @@ process.
 
 Total CPU seconds broken down by states.
 
-| Units | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
-|-------|-------------------|------------|------------------|------------------|
-|  `s`  | ObservableCounter | `Double`   | state            | user, system     |
+| Units | Instrument Type   | Value Type | Attribute Key(s)  | Attribute Values |
+|-------|-------------------|------------|-------------------|------------------|
+|  `s`  | ObservableCounter | `Double`   | process.cpu.state | user, system     |
```

Comment:
Not a blocking comment but spec defines this as Counter and we have it as ObservableCounter. I do not know the history of this, @Yun-Ting any idea?